### PR TITLE
redhat bundle image create files add

### PR DESCRIPTION
**Pull request by barakda**

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_3_0/bundle-1.3.0.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+
+COPY 1.3.0/manifests /manifests/
+COPY 1.3.0/metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.5,v4.6"
```
Comment (oriyarde) [inline deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_3_0/bundle-1.3.0.Dockerfile:12]: does this mean 1.3.0 was available on ocp 4.6?
is 1.4.0 available for ocp 4.4?

File: deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_4_0/1.4.0/metadata/annotations.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: stable
+  operators.operatorframework.io.bundle.channels.v1: stable
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: ibm-block-csi-operator
```
Comment (oriyarde) [inline deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_4_0/1.4.0/metadata/annotations.yaml:7]: could we change the order of these to match the dockerfile labels order? just to ease future comparisons

File: deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_3_0/bundle-1.3.0.Dockerfile (hunk 1 above)
Comment (barakda) [reply deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_3_0/bundle-1.3.0.Dockerfile:12]: - 1.3.0 is no longer available on ocp 4.6, the first publish has been done by red-hat team was wrong but it was fixed 
- yes 1.4.0 available for ocp 4.4, the new bundle project is relevant from version 4.5 (4.4 is published by the old way..)

File: deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_4_0/1.4.0/metadata/annotations.yaml (hunk 2 above)
Comment (barakda) [reply deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_4_0/1.4.0/metadata/annotations.yaml]: done - Dockerfile files have been updated

File: deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_3_0/bundle-1.3.0.Dockerfile (hunk 1 above)
Comment (oriyarde) [reply deploy/olm-catalog/ibm-block-csi-operator/rh_bundle/ibm-block-csi-operator-bundle_1_3_0/bundle-1.3.0.Dockerfile]: got it! thanks

**Review by oriyarde — COMMENTED** [review]
could you also remove the non-package yaml files from `ibm-block-csi-operator/deploy/olm-catalog/ibm-block-csi-operator/`?

**Review by oriyarde — APPROVED** [review]
LGTM

**Pull request merged**